src/grasshopper/lib/util/shapes.py

```diff
@@ -196,3 +197,52 @@ def __init__(self, *args, **kwargs):
             },
         ]
         super().__init__(*args, **kwargs)
+
+class Customstages(Default):  # noqa E501
+    """
+    Stolen from this set of examples as part of the locust.io documentation.
+    https://github.com/locustio/locust/blob/master/examples/custom_shape/stages.py
+
+    Keyword arguments:
+        stages -- A list of dicts, each representing a stage with the following keys:
+            duration -- When this many seconds pass the test is advanced to the next
+            stage.
+            users -- Total user count
+            spawn_rate -- Number of users to start/stop per second
+            stop -- A boolean that can stop that test at a specific stage
+        stop_at_end -- Can be set to stop once all stages have run.
+
+    Most likely, you'd want to extend this class and only define a new stages attr.
+    """
+
+    stages = [{"duration": 68, "users": 4, "spawn_rate": 0.5},
```

Comment:
you can feel free to have the default for this class be much simpler if you want, the default on the original was only there to provide an example

Reply:
Updated